app/assets/javascripts/application.js

```diff
@@ -12,5 +12,6 @@
 //
 //= require jquery
 //= require jquery_ujs
+//= require bootstrap-sprockets
 //= require jasny-bootstrap
```

Comment:
Let's keep this in and remove if/when we pull the old template out. Embedded still depends on this. Will need to separate out the styles for embedded and new app later. That way we aren't loading styles we are not using.